source/transfer_list.rst

```diff
@@ -951,6 +951,48 @@ software running in Secure, Non-Secure, or Realm modes.
      - hdr_size
      - Holds a single `entry_point_info` structure.
 
+**FF-A SP binary (XFERLIST_FFA_SP_BINARY)**
+
+This entry holds a reference to an FF-A secure partition's binary.
+
+Hafnium SPMC supports the SP's artefacts to be packaged as a Transfer List
+generated at build time (with TF-A's tlc tool). The respective Transfer List
+contents: SP binary/image, the FF-A manifest, and other optionl resoures
+(e.g. HOB).
```

Comment:
I think this whole paragraph should be in the commit message, not the spec itself. Also, you could say that there are already TEs defined for FF-A manifest and HOB list, but not for an SP binary.